README.rst

```diff
@@ -31,11 +31,12 @@ Endpoint                                 Description
 `/gzip`_                                 Returns gzip-encoded data.
 `/deflate`_                              Returns deflate-encoded data.
 `/status/:code`_                         Returns given HTTP Status code or random if more than one are given.
-`/response-headers`_                     Returns given response headers.
+`/response-headers?key=val`_             Returns given response headers.
 `/redirect/:n`_                          302 Redirects *n* times.
 `/redirect-to?url=foo`_                  302 Redirects to the *foo* URL.
 `/redirect-to?url=foo&status_code=307`_  307 Redirects to the *foo* URL.
 `/relative-redirect/:n`_                 302 Relative redirects *n* times.
+`/absolute-redirect/:n`_                 302 Absolute redirects n times.
```

Comment:
Please add the appropriate emphasis in the description to match the style of the rest of this document.